app/utils/constants.ts

```diff
@@ -86,10 +86,38 @@ async function getOpenAILikeModels(): Promise<ModelInfo[]> {
  }
 
 }
+
+async function getAzureOpenAIModels(): Promise<ModelInfo[]> {
+  try {
+    const base_url = import.meta.env.AZURE_OPENAI_API_BASE_URL || "";
+    if (!base_url) {
+      return [];
+    }
+    const api_key = import.meta.env.AZURE_OPENAI_API_KEY;
+    const response = await fetch(`${base_url}/openai/deployments?api-version=2023-03-15-preview`, {
```

Comment:
@HuynhDoTanThanh would it make sense to make the API Version parametrizable in `.env` as well?

Reply:
Hell @HuynhDoTanThanh,

having same issue than @axen2u. It's using anthropic instead of azure url.

can you please check ?